### PR TITLE
Set up Feedback messages for Rabbit and Slack

### PR DESCRIPTION
**Pull request by anandaroop**
This follows the pattern for existing topics and adds the configuration for "Feedback" messages coming from Gravity.

(This also includes some documentation updates about things we fumbled through while setting up the project)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+SLACK_API_TOKEN=REPLACE
```
Comment (ashkan18) [inline .env.example]: Thanks!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,20 +24,31 @@ Install Elixir, see [installation](http://elixir-lang.org/install.html).
...
+mix ecto.create
+mix ecto.migrate
 ```
 
+Start the app:
+
+```sh
+mix run --no-halt
+# or, if you want to enable pry debugging
+iex -S mix run --no-halt
```
Comment (ashkan18) [inline CONTRIBUTING.md]: Thaaaaanks! 🙌

File: lib/views/feedbacks_slack_view.ex [hunk 3]
```diff
@@ -0,0 +1,8 @@
+defmodule Aprb.Views.FeedbacksSlackView do
+
+  def render(event) do
+    %{
+      text: "#{event["subject"]["display"]} #{event["verb"]} #{event["properties"]["message"]} from #{event["properties"]["url"]}",
```
Comment (ashkan18) [inline lib/views/feedbacks_slack_view.ex]: Did you try this new view? We read `attachments` and `unfurl_links` from this render response [here](https://github.com/artsy/aprb/blob/master/lib/services/event_service.ex#L15) when calling slack, both of them will be `nil` in this view, i don't know how Slack api deals with `nil` for both of these. If they don't handle it well, we can set defaults for those two fields.

Comment (anandaroop) [inline lib/views/feedbacks_slack_view.ex]: Oh good to know. 

We couldn't figure out getting local APRb to talk to staging Rabbit, so maybe we can try now that all the PRs have been merged to staging.

Comment (joeyAghion) [inline lib/views/feedbacks_slack_view.ex]: Careful connecting a local consumer to staging RabbitMQ, as I think there have been problems disconnecting these later on when a real staging consumer wants to step in.

@ashkan18 did I remember this correctly? What's the recommended practice around different environments and their bindings?

Comment (ashkan18) [inline lib/views/feedbacks_slack_view.ex]: yes, basically there are two things we need to do when connecting locally to staging RabbitMQ, we need to first change the queue name locally to something like `"ashkan_local_aprb_#{topic}_queue" [here](https://github.com/artsy/aprb/blob/master/lib/services/amq_event_service.ex#L24) so we don't consume staging events. This then will create a new queue in RabbitMQ staging, the issue is it won't delete the queue once we stop the connection, so we need to manually go to dashboard and delete the queue. I _think_ there are few options we can use to delete the queue if nothing is connected but that may not make sense for staging/production cases.

**Comment by joeyAghion** [comment]
I'm wondering if we could eliminate the need for APRB to change with every new topic/event we care about. Could Slack users just subscribe to a given topic and routing key expression, with APRB starting an associated worker if necessary? (Not a swap suggestion, just thinking...)

**Comment by ashkan18** [comment]
that would be ideal, i think in general it makes sense to allow to subscribe to specific verb/routing key eventually.
We do have a table of all existing Topics, technically we could just get all topics from that table and have APRB subscribe to each one on RabbitMQ, but still for any new topics we need to handle how we want to show the slack message. Unless we come up with a generic message format.

**Pull request merged**